js/loadThePilesOfCards.js

```diff
@@ -0,0 +1,105 @@
+function feelTheField() {
+	
+}
+function shuffle(a) {
+	var j, x, i;
+	for (i = a.length; i; i -= 1) {
+		j = Math.floor(Math.random() * i);
+		x = a[i - 1];
+		a[i - 1] = a[j];
+		a[j] = x;
+	}
+}
+document.body.onselectstart= function() {return false};
+shuffle(arrayOfCards);
+var hidenCardsDeck = arrayOfCards.splice(0,24);
+var openCardsDeck=[];
+var NUMBERoFpILES = 7;
+
+
+for(var i=0;i<NUMBERoFpILES;i++)
+{
+    var nowPileDiv=document.getElementById(i+1);
+    for(var j=0;j<i;j++)
```

Comment:
I do not think that it is very good idea to use id's to find piles. Markup may change and if your game were only one of the widgets on the page it would probably break down if other widgets used the same id's. As I know the general advice is to use id only for the root element like container div...